lib/dns/resource.js

```diff
@@ -1203,10 +1203,12 @@ class Resource extends Struct {
       case types.A:
         res.answer = this.toA(name);
         key.signZSK(res.answer, types.A);
+        key.signZSK(res.answer, types.TXT);
```

Comment:
Without context, this code block may be strange to a reader. What do you think about adding a descriptive guard, e.g.:
```js
case types.A:
  res.answer = this.toA(name);
  key.signZSK(res.answer, types.A);
  if (this.hasTor())
    key.signZSK(res.answer, types.TXT);
  break;
```